### PR TITLE
[feature] Contributions

### PR DESCRIPTION
**Pull request by cazala**
List contributions on the sidebar

### CODE REVIEW

File: webapp/src/components/HomePage/Sidebar/ExpandedSidebar/WalletContributions/ContributionRow/ContributionRow.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    const { contribution } = this.props
+
+    return (
+      <div className="table-row">
+        <div className="col col-district">
+          <DistrictLink contribution={contribution} />
+        </div>
+        <div className="col col-contributed">
+          {contribution.land_count.toLocaleString()} LAND ({contribution.land_count *
+            1000}{' '}
```
Comment (NicoSantangelo) [inline webapp/src/components/HomePage/Sidebar/ExpandedSidebar/WalletContributions/ContributionRow/ContributionRow.js]: What do think about adding an object to handle this type of conversions? Maybe we could have a `land` domain similar to what we had here: https://github.com/decentraland/auction/blob/master/webapp/src/lib/land.js

so we can do `land.convert(contribution.land_count, 'mana')`

File: webapp/src/components/HomePage/Sidebar/ExpandedSidebar/WalletParcels/ParcelTable/ParcelTable.js [hunk 2]
```diff
@@ -26,9 +26,7 @@ class ParcelTable extends React.PureComponent {
         </div>
       )
     } else {
-      return (
-        <div className="table-row-empty">You have no available land yet</div>
-      )
+      return <div className="table-row-empty">{"You don't own any land."}</div>
```
Comment (NicoSantangelo) [inline webapp/src/components/HomePage/Sidebar/ExpandedSidebar/WalletParcels/ParcelTable/ParcelTable.js]: Why `{}` here? ( for `{"You don't own any land."}` )

File: webapp/src/modules/address/sagas.js [hunk 3]
```diff
@@ -38,3 +45,31 @@ function* handleAddressParcelsSuccess(action) {
     parcels: action.parcels
   })
 }
+
+function* handleAddressContributionsRequest(action) {
+  try {
+    const { address } = action
+    let contributions = yield call(() => api.fetchAddressContributions(address))
+
+    contributions = [
```
Comment (NicoSantangelo) [inline webapp/src/modules/address/sagas.js]: I think we should remove this before merging into `master` don't we?

File: webapp/src/components/HomePage/Sidebar/ExpandedSidebar/WalletContributions/WalletContributions.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import ContributionTable from './ContributionTable'
+import Loading from 'components/Loading'
```
Comment (NicoSantangelo) [inline webapp/src/components/HomePage/Sidebar/ExpandedSidebar/WalletContributions/WalletContributions.js]: Unused var!

**Comment by NicoSantangelo** [comment]
screen shot 2018-01-14 at 12 49 04

What do you think about adding a third column `proposal` with the link to the github proposal?

**Comment by abarmat** [comment]
Maybe some icon with the link to the proposal?

**Comment by cazala** [comment]
like dis?
screen shot 2018-01-14 at 9 10 34 pm

File: webapp/src/components/HomePage/Sidebar/ExpandedSidebar/WalletParcels/ParcelTable/ParcelTable.js (hunk 2 above)
Comment (cazala) [reply webapp/src/components/HomePage/Sidebar/ExpandedSidebar/WalletParcels/ParcelTable/ParcelTable.js]: it was cos of the single quote :P but yea I can alwas `&apos;`

**Pull request merged**